Makefile

```diff
@@ -29,7 +29,8 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # codeflare.dev/instaslice-operator-bundle:$VERSION and codeflare.dev/instaslice-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= quay.io/amalvank/instaslicev2
+IMAGE_TAG_BASE ?= quay.io/openshift/instaslicev2
```

Comment:
/hold
Until we decide on this.
Not sure if this is a valid place to store the images.
cc: @rphillips